cclib/parser/turbomoleparser.py

```diff
@@ -65,6 +65,7 @@ def normalisesym(self, label):
 
     def before_parsing(self):
         self.geoopt = False # Is this a GeoOpt? Needed for SCF targets/values.
+        self.scf_converged = True # Need a check to determine whether scf convergence fails.

```

Comment:
Shouldn't this be set to `False` here and be changed to `True` when it has converged?

Reply:
I am trying to find a trigger for finding if the convergence occurs but the problem is that Turbomole prints energy converged before the last iteration. I was thinking of parsing the data as is and change this to `true` and stop iterating when we find the `energy converged` keyword (something along these lines)